configuration/code_schemes.py

```diff
@@ -24,6 +24,7 @@ class CodeSchemes(object):
     S01E10 = _open_scheme("s01e10.json")
     S01E11 = _open_scheme("s01e11.json")
     S01E12 = _open_scheme("s01e12.json")
+    DIAGNOSTIC_S01E03 = _open_scheme("diagnostic_s01e03.json")
```

Comment:
I wonder if it makes for this line to be next to the other diagnostics. It depends whether this is more useful to be organised by type or by time.

Reply:
by type is better - updated thanks